classes/03class/exercises/c03-compose01/gpavelar/docker-compose.yaml

```diff
@@ -0,0 +1,27 @@
+version: "3.0"
+services:
+  db:
+    image: mysql:5.7
+    restart: always
+    environment:
+      MYSQL_DATABASE: devops_db
+      MYSQL_ROOT_PASSWORD: password
+    ports:
+      - "3306:3306"
+    expose:
+      - "3306"
```

Comment:
Just a comment here, if we would like that external client to access the database, we would specify the ports here. Otherwise, we would let only services inside the docker virtual network to access the database and not specify the ports here.

The `expose` option here, it is used when the image doesn't specify which port its service exposes. For the MySQL image, the maintainer already added that so you don't need to specify it here.

You can check it here: https://github.com/docker-library/mysql/blob/a6b87c55ecdf9426d42ff7814be7e9b1fc33b292/5.7/Dockerfile#L89

Reply:
Great! I did not know that it could be a future problem.